packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -697,7 +703,7 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
 
         periodicLogger.stop();
         // eslint-disable-next-line max-len
-        await client.setStatusMessage?.(`Finished! Total ${this.stats.state.requestsFinished + this.stats.state.requestsFailed} requests: ${this.stats.state.requestsFinished} succeeded, ${this.stats.state.requestsFailed} failed.`, { isStatusMessageTerminal: true });
+        await this.setStatusMessage(`Finished! Total ${this.stats.state.requestsFinished + this.stats.state.requestsFailed} requests: ${this.stats.state.requestsFinished} succeeded, ${this.stats.state.requestsFailed} failed.`, { isStatusMessageTerminal: true });
```

Comment:
i am not entirely sure we should be setting the `isStatusMessageTerminal` flag here, it was meant to be set via `Actor.exit/fail/abort`, but those usually go after the crawler resolves. 

@mhamas is it allowed to do the call twice in a row with `isStatusMessageTerminal: true`? if so it sounds good to me to use it in the final message from crawler too

Reply:
Yes, setting it twice should work.